### PR TITLE
Adding the AddParentEmailModal

### PR DESCRIPTION
**Pull request by bethanyaconnor**
First PR for the modal to add a parent email -- I'll follow up with a second one to add the controller. Screenshots below from Storybook so ignore the dark grey background.

[spec](https://docs.google.com/document/d/1iOfV0Opkg9Jqw3LEmKtmq4KCWbmeCnTkSYvf-_BNYIg/edit?ts=5e8373a9#heading=h.6jx150u3fizm)

On load:
Screenshot 2020-04-01 at 12 11 23 PM

Invalid email:
Screenshot 2020-04-01 at 12 11 31 PM

Valid inputs:
Screenshot 2020-04-01 at 12 11 42 PM

Saving:
Screenshot 2020-04-01 at 12 11 48 PM

## Links

- [spec](https://docs.google.com/document/d/1iOfV0Opkg9Jqw3LEmKtmq4KCWbmeCnTkSYvf-_BNYIg/edit?ts=5e8373a9#heading=h.6jx150u3fizm)
- [jira](https://codedotorg.atlassian.net/browse/FND-1094)

## Testing story
- Unit tests
- Storybook

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/ui/accounts/AddParentEmailModal.jsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+            <div style={styles.emailOptIn}>
+              <label style={styles.label}>
+                {i18n.addParentEmailModal_emailOptIn_description()}{' '}
+                <a href={pegasus('/privacy')}>
+                  {i18n.changeEmailModal_emailOptIn_privacyPolicy()}
+                </a>
+              </label>
+              <div>
+                <div style={styles.radioButton}>
+                  <label style={styles.input}>
```
Comment (Hamms) [inline apps/src/lib/ui/accounts/AddParentEmailModal.jsx]: why do we have labels without text, and why are the labels with the text not associated with their respective inputs?

Comment (bethanyaconnor) [reply apps/src/lib/ui/accounts/AddParentEmailModal.jsx]: The labels without text are removed.

I'm not sure what you mean by the second bit.

Comment (Hamms) [reply apps/src/lib/ui/accounts/AddParentEmailModal.jsx]: Labels can (edit: and should) be explicitly associated with the inputs they are labeling to support screen readers and provide some interfacing assistance (clicking the label focuses the input, etc).

You can either wrap the input in the label, like

```html
<label>
  <input />
  {i18n.yes()}
</label>
```

or give the input an `id` and the label a matching `for` attribute, like

```html
<input id="yes" />
<label for="yes">{i18n.yes()}</label>
```

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label for more info

**Comment by dju90** [comment]
We updated the text of the email opt-in this morning: do you mind changing it to "Can we email you with occasional updates on your child’s progress and projects, and updates about their course and computer science? (See our privacy policy)"

File: apps/src/lib/ui/accounts/AddParentEmailModal.jsx (hunk 1 above)
Comment (bethanyaconnor) [reply apps/src/lib/ui/accounts/AddParentEmailModal.jsx]: Thanks! I used the `htmlFor` attribute (React's version of `for`) for the labels

**Comment by bethanyaconnor** [comment]
New screenshot with updated copy:
Screenshot 2020-04-02 at 9 56 06 AM

**Review by Hamms — APPROVED** [review]

**Review by daynew — APPROVED** [review]
LGTM but I'm no react expert!

**Pull request merged**